docs/index.md

```diff
@@ -44,36 +68,54 @@ can be used to:
-        -DUNICODETOOLS_REPO_DIR=/usr/local/google/home/mscherer/unitools/mine/src
-        -DUNICODETOOLS_OUTPUT_DIR=/usr/local/google/home/mscherer/unitools/mine
-        -DUVERSION=14.0.0
-        ```
-    4.  Please also use the VM argument `-ea` (enable assertions) in your Preferences
-        or in your Run/Debug configurations, so that failed assertions don’t just slip through.
 
-### Command Line Instructions
+### Running commands for Unicode Tools tasks
 
-1. Set up Maven per <http://cldr.unicode.org/development/maven> (You will need CLDR checked out to run tools)
+#### Setting system properties
 
-2. Here are some example maven command lines:
+For the tools to work, you need to set the JVM system properties according to your workspace layout.
 
-```shell
-# assuming CLDR is ../cldr and output dir is ../Generated
+For command-line users:
+- System properties are specified in this fashion for Maven (same as it is for the JVM CLI): `-Dvar1=path1 -Dvar2=path2 ...`
+- Depending on which tool you are running, you may need some or all of the properties listed above in General Setup for Maven. Adjust the path values to point to your workspace directory locations. Ex:
+```
+-DCLDR_DIR=/usr/local/google/home/mscherer/cldr/uni/src
+-DIMAGES_REPO_DIR=/usr/local/google/home/mscherer/images/mine/src
+-DUNICODETOOLS_REPO_DIR=/usr/local/google/home/mscherer/unitools/mine/src
+-DUNICODETOOLS_OUTPUT_DIR=/usr/local/google/home/mscherer/unitools/mine
+-DUVERSION=14.0.0
+```
+
+For Eclipse users:
+- You can set these for each single tool in the Run|Debug Configurations... (x)= Arguments tab, in the VM arguments
+- or you can set the common variables globally in Window > Preferences... > Java > Installed JREs, select the active JRE, Edit... Default VM arguments: `-Dvar1=path1 -Dvar2=path2 ...`. For this alternative, see the example above for command-line users.
+
+#### Enabling assertions
+
+Please also enable assertions when running commands so that failed assertions don’t just slip through.
+
+Command-line users:
+- Set the `MAVEN_OPTS` environment variable to include the `-ea` JVM option in its string value
+  * Ex: `export MAVEN_OPTS="-ea"; mvn exec:java -Dexec.mainClass=...`
+  * Ex: `MAVEN_OPTS="-ea" mvn exec:java -Dexec.mainClass=...`
+
+Eclipse users:
+- Use the VM argument `-ea` (enable assertions) in your Preferences or in your Run/Debug configurations
 
+### Commands for Unicode Tools Tasks
 
-# MakeUnicodeFiles
-mvn exec:java -Dexec.mainClass="org.unicode.text.UCD.Main" -Dexec.args="version 14.0.0 build MakeUnicodeFiles" -pl unicodetools  -DCLDR_DIR=$(cd ../cldr;pwd) -DUNICODETOOLS_OUTPUT_DIR=$(cd ..;pwd) -DUNICODETOOLS_REPO_DIR=$(pwd) -DUVERSION=14.0.0
-# build and test
-mvn package -DCLDR_DIR=$(cd ../cldr;pwd) -DUNICODETOOLS_OUTPUT_DIR=$(cd ..;pwd) -DUNICODETOOLS_REPO_DIR=$(pwd) -DUVERSION=14.0.0
+Common tasks for Unicode Tools are listed below with example CLI commands with example argument values that they need:
```

Comment:
What are all the basic unicodetools tasks and corresponding CLI command examples that should be listed here?